src/wallet/components/WaitingStep.tsx

```diff
@@ -15,9 +15,11 @@ export default class WaitingStep extends React.PureComponent<Props> {
       <div className={css(styles.container)}>
         <div>
           <h1>
-            Waiting for {message}
-            ...
+            Wallet Screen
```

Comment:
I'm confused about this change. If this component is _called_ WaitingStep, it should probably involved something about Waiting, rather than just rendering the text `Wallet Screen`. If it's no longer specifically about waiting, then the name of the component should be changed.